### ISSUE TITLE
is it possible to add "How to use" widget

### ISSUE THREAD

**Issue by BarbaraTempl** [ISI-MIP/isipedia-text-generator#13]
There is a paragraph about "How to use the interactive figure" (in the drought.md).
It would be nicer if it is not embedded in the text of the report (as it is now) but added like an extra box with text - usually also the "Did you know?" texts are extra info which are addded to the main text and the user read it only if he/she is interested in it. What do you think? Robert can do it?

**Comment by perrette** [comment]
I do not see the paragraph !

**Comment by perrette** [comment]
ok in drought_world.md

**Comment by perrette** [comment]
I agree it is strange to have it there, or in any case there should be only one such text box for all reports, not one for each. We can think together how best to display it, but my suggestion for now is to use a smarter templating system where everything that is common should be written once and for all, and "imported" or inserted where we need it (e.g. in every report). As a basic principle, it is important to remove redundancy otherwise the maintainance is too heavy.

@BarbaraTempl  if you are interested take a look at [jinja2's template inheritance system](https://jinja.palletsprojects.com/en/2.10.x/templates/#template-inheritance) -- I would program it, no worry, just to understand the philosophy.

**Comment by perrette** [comment]
I mean the above also for world & country report, and even certain some section common to all indicators. Otherwise the various text will drift away from each other (country and world-level, various indicators). Let's us elaborate a smart templating with inheritance mechanism.

**Comment by perrette** [comment]
Let us have this discussion in Issue #12

**Comment by perrette** [comment]
@matthiasmengel  can you add Robert to all these repos?
